### ISSUE TITLE
Are the linkouts getting a bit GCV-redundant?

### ISSUE THREAD

**Issue by sammyjava** [legumeinfo/microservices#612]
I was looking at gene linkouts and realized that I don't know what the difference is between the three GCV linkouts. I understand that they're probably different, a difference which is understood by a few people on Earth, but do we want to make users figure out which one to click, or, rather, choose one that makes the most sense for the "typical user" (that doesn't work for LIS)?

![image](https://github.com/legumeinfo/microservices/assets/5657219/78dd748f-1c9e-49b9-8f02-37aea6a038ee)

**Comment by adf-ncgr** [comment]
I agree that this probably should be simplified, but as an explanation of the differences we have:
1) a gene-based link to the "main GCV" which contains more or less one representative genome per species (this link will only be present for "representative genomes", ie for glyma.Wm82 but not glyma.Lee)
2) a gene-based link to the "genus GCV", which contains only genomes for that genus (glyma.Wm82 and glyma.Lee but not medtr)
3) a region-based link to the "genus GCV"; I would actually like to get rid of this one because it takes you to a pretty useless view with a gene neighborhood of one, but we need to link genes out as regions for the sake of getting to JBrowse2, and the region-based linkout isn't smart enough to tell the difference (or know that a gene-based linkout call is being made). I could imagine some extra parameter hack that would allow us to inform the region linkout that it's not a multi-gene region (or something like that) which would make it non-interesting for things like GCV.

I should add that we are also releasing pangene-level GCVs, which would be only genus-level but would potentially add yet another GCV choice to linkouts. So it's definitely a good time to decide how to make this all more "typical user"-friendly!

**Comment by sammyjava** [comment]
The explanation isn't for me, it's for our users. I am aware that there are differences. But a random user has no idea how those links differ.

**Comment by adf-ncgr** [comment]
Yes, I thought you might have suggestions on how to rephrase the link texts to make the distinctions clearer (without being overly verbose)

**Comment by sammyjava** [comment]
> Yes, I thought you might have suggestions on how to rephrase the link texts to make the distinctions clearer (without being overly verbose)

Ahhhh, no, I'll leave that for you. :) I think we should just link one Mother of All GCVs.

**Comment by adf-ncgr** [comment]
I guess that would be analogous to what we currently to for mine linkouts, ie only take people to LegumeMine, the Mother of all Mines. But you're tempting me to add links to the genus mines too so that the GCV weight is at least counterbalanced...
